src/Stateless/ActivateActionBehaviour.cs

```diff
@@ -5,64 +5,34 @@ namespace Stateless
 {
     public partial class StateMachine<TState, TTrigger>
     {
-        internal abstract class ActivateActionBehaviour
+        internal class ActivateActionBehaviour
         {
             readonly TState _state;
+            private readonly EventCallback _callback;
```

Comment:
use EventCallback is not enought good solution. library can used in simple console host application that does not have this dependency